_data/mining.yml

```diff
@@ -35,13 +35,35 @@ websites:
       bch: yes
       btc: no
       othercrypto: no
+      
+    - name: GPU Mining Bios
+      url: https://mining-bios.eu/
+      img: miningbios.png
+      facebook: indeedminers
+      region: eu
+      bch: yes
+      btc: yes
+      othercrypto: yes
+      doc: https://mining-bios.eu/2018/01/21/more-native-crypto/
 
     - name: Suprnova
       url: https://www.suprnova.cc/
       img: suprnova.png
       bch: Yes
       btc: Yes
       othercrypto: Yes
+      
+    - name: Taimine
+      url: https://taimine.com
+      img: taimine.png
+      twitter: taimineminer
+      facebook: taimineindustries
+      email_address: service@taimine.com
+      region: as
+      city: all of asia
```

Comment:
Can probably just omit the city tag here, but I'm not that picky

Reply:
done

Reply:
@joshmcintyre good catch, Fixed and ready to go!

After we merge please take a look over the rest of the issues. I would've done them but I wasnt sure the categories they should fall into.